client/src/components/Reviews/components/ReviewList.jsx

```diff
@@ -38,19 +39,27 @@ const StyledSelect = styled.select`
 
 function ReviewList() {
   const [currentSort, setCurrentSort] = useState('relevant');
+  const [toggleModal, setToggleModal] = useState(false);
```

Comment:
I would suggest using verbs for function names and [prefixing boolean variables with something like "is" or "has" or "can"](https://www.samanthaming.com/tidbits/34-better-boolean-variable-names/). For example, you could use "isModalActive". The boolean prefixing is maybe not as important, but I do think it's valuable to reserve verb prefixes like "toggle" for functions. It helps provide an implicit description for your variables.